packages/pancake-uikit/src/components/Box/Grid.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+import { grid } from "styled-system";
+import Flex from "./Flex";
+import { GridProps } from "./types";
+
+const Grid = styled(Flex)<GridProps>`
```

Comment:
Why use `<Flex />` as the base and not `<Box />`?

Reply:
https://styled-system.com/api#flexbox

`justifyItems` and other alignments props are part of the flex helper